frontend/src/app/libs/auth/services/auth.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { User } from '../models/user';
+
+export const httpOptions: {} = {
+	headers: new HttpHeaders({
+		'Content-Type': 'application/json'
+	})
+};
```

Comment:
lets create http interceptor for purpose of setting content type header to any http request.